Deserialization Error on BandPlan

```
20200804 203705.146 [sdrtrunk pool-1-thread-11] ERROR i.g.d.g.p.r.SystemEditor - Error retrieving system information  [97MB/150MB 64%]
io.github.dsheirer.rrapi.RadioReferenceException: Error deserializing XML response: [SOAP envelope snipped]
	at io.github.dsheirer.rrapi.RadioReferenceService.submitSync(RadioReferenceService.java:1208)
	at io.github.dsheirer.rrapi.RadioReferenceService.getSites(RadioReferenceService.java:456)
	at io.github.dsheirer.service.radioreference.CachingRadioReferenceService.access$1101(CachingRadioReferenceService.java:44)
	at io.github.dsheirer.service.radioreference.CachingRadioReferenceService.getSites(CachingRadioReferenceService.java:203)
	at io.github.dsheirer.gui.playlist.radioreference.SystemEditor.lambda$setSystem$5(SystemEditor.java:282)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "item" (class io.github.dsheirer.rrapi.type.Bandplan), not marked as ignorable (3 known properties: "base", "offset", "spacing"])
 at [Source: (StringReader); line: 1, column: 4282] (through reference chain: io.github.dsheirer.rrapi.response.ResponseEnvelope["Body"]->io.github.dsheirer.rrapi.response.GetSitesResponse["return"]->java.util.ArrayList[2]->io.github.dsheirer.rrapi.type.Site["bandplan"]->io.github.dsheirer.rrapi.type.Bandplan["item"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:823)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1153)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1589)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1567)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:294)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:127)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:286)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:245)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:27)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:127)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
	at com.fasterxml.jackson.databind.jsontype.impl.AsWrapperTypeDeserializer._deserialize(AsWrapperTypeDeserializer.java:121)
	at com.fasterxml.jackson.databind.jsontype.impl.AsWrapperTypeDeserializer.deserializeTypedFromObject(AsWrapperTypeDeserializer.java:52)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeWithType(BeanDeserializerBase.java:1178)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:136)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4013)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3004)
	at io.github.dsheirer.rrapi.RadioReferenceService.deserialize(RadioReferenceService.java:1227)
	at io.github.dsheirer.rrapi.RadioReferenceService.submitSync(RadioReferenceService.java:1193)
	... 10 common frames omitted
```

--- a/src/main/java/io/github/dsheirer/rrapi/type/SystemInformation.java
+++ b/src/main/java/io/github/dsheirer/rrapi/type/SystemInformation.java
@@ -19,6 +19,7 @@
 
 package io.github.dsheirer.rrapi.type;
 
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
 
 import java.util.ArrayList;
@@ -297,7 +298,8 @@ public class SystemInformation
      * List of radio system identifiers
      * @return system id listing
      */
-    @JacksonXmlProperty(localName = "sysid")
+    @JacksonXmlElementWrapper(localName = "sysid")
+    @JacksonXmlProperty(localName = "item")
     public List<RadioNetwork> getRadioNetworks()
     {
         return mRadioNetworks;

